apps/financial-aid/web-veita/src/components/Profile/MunicipalityProfile.tsx

```diff
@@ -216,7 +269,7 @@ const MunicipalityProfile = ({
                     identifier={value}
                     key={`aidTableBody-${value}`}
                     hasMaxWidth={false}
-                    animationDelay={55 * (municipality.adminUsers?.length ?? 1)}
+                    animationDelay={55 * index}
```

Comment:
Is this necessary, since we have  style={{ animationDelay: animationDelay + 3.5 * index + 'ms' }} in the component? 👯‍♂️